pypeit/manual_extract.py

```diff
@@ -96,15 +103,22 @@ def by_fitstbl_input(cls, frame: str, inp: str, spectrograph):
             idict['spat'] += [float(parse[1])]
             idict['spec'] += [float(parse[2])]
             idict['fwhm'] += [float(parse[3])]
+            # Boxcar?
+            if len(parse) >= 5:
+                idict['boxcar_rad'] += [float(parse[4])]
+            else:
+                idict['boxcar_rad'] += -1.
```

Comment:
Should this be `idict['boxcar_rad'] += [-1.]`; i.e., you're appending to the list not adding a value, right?

Reply:
yeeps, good catch!

Reply:
Added a test to catch this from here on.